test/e2e/argo_server_test.go

```diff
@@ -623,7 +626,33 @@ func (s *ArgoServerSuite) TestPermission() {
 			Status(403)
 	})
 
+	// Test get wf w/ archive fallback with good token
+	s.bearerToken = goodToken
+	s.Run("GetWFsFallbackArchivedGoodToken", func() {
+		s.e().GET("/api/v1/workflows/"+nsName).
```

Comment:
nit/incidental finding: so the spacing around the `+` is different on this line than on line 641 and 649 below. I would think `gofmt` would make these consistent though -- it didn't make changes in https://github.com/argoproj/argo-workflows/pull/12979#discussion_r1578646195 either so I was thinking it's possibly not running on test files? Afaik `gofmt` usually does `x + y` with a space on both sides of the `+`

See also https://github.com/golang/go/issues/12105 -- only one argument here

Reply:
Yeah, this is pretty bonkers, and I don't understand why. I haven't analysed it any way because I was concentrating on getting the release out.

CI does pick it up, as you can see [here](https://github.com/argoproj/argo-workflows/actions/runs/12121071524/job/33791245905) when I pushed it up initially with the fixed formatting. The [original test](https://github.com/argoproj/argo-workflows/blob/06c761b8cc993aa6ab60f8c35c3c95bb334f3da0/test/e2e/argo_server_test.go#L611) has the same thing, so the linter had 'formatted' it there too when you wrote it.

I'm most thrown by the fact the other two lines you pointed out are formatted correctly, it's just this one that's "wrong", yet they are all so similar.

Reply:
> I haven't analysed it in any way because I was concentrating on getting the release out.

For sure 👍 
I just noticed it incidentally

> CI does pick it up, as you can see [here](https://github.com/argoproj/argo-workflows/actions/runs/12121071524/job/33791245905) when I pushed it up initially with the fixed formatting. The [original test](https://github.com/argoproj/argo-workflows/blob/06c761b8cc993aa6ab60f8c35c3c95bb334f3da0/test/e2e/argo_server_test.go#L611) has the same thing, so the linter had 'formatted' it there too when you wrote it.

Well that's more bizarre than I thought 👀 Thanks for checking that!
In this case I'd hazard a guess this might be a `gofmt` bug or some fairly odd quirk explained by some other line widths; potentially worth filing a bug report in either scenario 😅